modules/appeals_api/spec/support/schema_helpers.rb

```diff
@@ -15,4 +15,19 @@ def read_schema(filename, schema_version = 'v1')
       )
     )
   end
+
+  def read_schema_v2(filename, schema_version = 'v2')
```

Comment:
helper method to point to the appropriate schema file to test against for the schema spec.

Reply:
Why are we passing in the schema version as an argument if the method name is already constrained to the version?

Reply:
The method name doesn't constrain it to the version...the schema version variable is passed into the path that the method reads in order to pull the schema from the correct directory/file. Am I missing something?

Reply:
Never mind. I see lol.